bounties_api/std_bounties/management/commands/bounties_subscriber.py

```diff
@@ -64,61 +70,211 @@ def handle(self, *args, **options):
                     'attempting {}: for bounty id {}'.format(
                         event, str(bounty_id)))
                 if event == 'BountyIssued':
-                    bounty_client.issue_bounty(
-                        bounty_id, contract_method_inputs, event_timestamp)
+                    msg = "{title}, {bounty_id}, {tokenSymbol} @ {tokenDecimals}, {fulfillmentAmount}, "\
+                          "{usd_price}, {deadline} {link}"
+                    bounty = Bounty.objects.filter(bounty_id=bounty_id)
+                    add_link = partial(merge, source2={'link': bounty_url_for(bounty_id)})
+
+                    if not bounty.exists():
+                        apply_and_notify(bounty_id,
```

Comment:
@zoek1 - so far looking good. This definitely adds complexity and a lot to read through on this file.  I think this would be cleaner for now if there was a client that included all this code and then also called the current client... that way this file can still read as, master_client.issue_bounty etc. etc. Make sense?

Reply:
Totally agree! consider it done.